atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ResultsExtractor.java

```diff
@@ -91,4 +100,9 @@ public abstract void internalExtractResult(long startTs,
                                                long ts);
 
     public abstract Map<Cell, T> asMap();
+
+    protected Meter getPostFilteredCellsMeter(Class clazz) {
+        // TODO(hsaraogi): add table names as a tag
+        return metricsManager.registerOrGetMeter(clazz, "startTsCellFilterCount");
```

Comment:
Would recommend making this a static package visible method that `return metricsManager.registerOrGetMeter(class, “postfilteredCells”);` as that provides more context and the meter itself captures count, 1m, 5m, 15m, mean

Reply:
Also reuse this from other places as needed

Reply:
`postfilteredCells` is a vague name as it does not capture the reason for the cells being filtered out. I refactored this to a separate class `AtlasDbMetricNames` and changed the name to `notLatestVisibleValueCellFilterCount` as that is the actual reason for them being discarded from the client..